src/app/api/user/registerUser/route.js

```diff
@@ -2,7 +2,7 @@ import { NextResponse } from 'next/server'
 import dbConnect from '../../../../../utils/dbConnect';
 import User from '../../../../../models/User';
 
-export const POST = async(req) => {
+export const POST = async(req) => { //needs be rewritten to handle transition to supabase

```

Comment:
PRAISE: Nice job being proactive with these funcs! This helps set up what we need to work on next!